tests/eosio.system_tester.hpp

```diff
@@ -1332,7 +1333,7 @@ class eosio_system_tester : public validating_tester {
                              )
          );
       }
-      produce_blocks( 250 );
+      produce_blocks( 2 * 21 ); // This is minimum number of blocks required by ram_gift in system_tests
```

Comment:
Can that be moved to `ram_gift` test?

Reply:
Other tests using `active_and_vote_producers()` (like producers_upgrade_system_contract and setparams) require 21. Probably not worth the effort to do it individually in each of its users.

Reply:
Now the longest test `eosio_system_inflation_unit_test` takes 103 seconds, a much improvement of 433 seconds for system_tests before the change (https://github.com/AntelopeIO/reference-contracts/actions/runs/9487310948/job/26143632160#step:7:978)